client/verta/verta/_internal_utils/time_utils/__init__.py

```diff
@@ -33,5 +36,30 @@ def epoch_millis(dt):
         raise ValueError("Cannot convert argument to epoch milliseconds")
 
 
+def timedelta_millis(delta):
+    if isinstance(delta, timedelta):
+        return int(delta.total_seconds() * 1000)
+    elif type(delta) == int and delta > 0:
+        return delta
+    else:
+        raise ValueError("Cannot convert argument to duration milliseconds")
```

Comment:
Since the other functions don't use an `else` for the catch-all error:
```suggestion
    raise ValueError("Cannot convert argument to duration milliseconds")
```